java/test/src/main/java/org/ray/api/test/FailureTest.java

```diff
@@ -14,12 +14,26 @@
 import org.ray.api.exception.RayWorkerException;
 import org.ray.api.function.RayFunc0;
 import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class FailureTest extends BaseTest {
 
   private static final String EXCEPTION_MESSAGE = "Oops";
 
+  @BeforeClass
+  public void setUp() {
+    // Set one worker per process. Otherwise, if `badFunc2` and `slowFunc` run in the same
+    // process, `sleep` will delay `System.exit`.
```

Comment:
Can you also document the name of the test case to be fixed?

Reply:
fixed